R/brmsfit-helpers.R

```diff
@@ -1019,15 +1070,37 @@ write_brmsfit <- function(x, file, compress = TRUE) {
   invisible(x)
 }
 
+
+
 # check validity of file name to store a brmsfit object in
-check_brmsfit_file <- function(file) {
+check_brmsfit_file_name <- function(file) {
   file <- as_one_character(file)
   file_ending <- tolower(get_matches("\\.[^\\.]+$", file))
   if (!isTRUE(file_ending == ".rds")) {
     file <- paste0(file, ".rds")
   }
   file
 }
+# Prepare and optionally validate a file path for a brmsfit object
+# @param file A character string specifying a file path or bare filename.
+# @param .check_folder Logical; if TRUE (default), verify that the folder exists via check_folder().
+#                      If FALSE, skip this existence check.
+# @return A normalized file path (character) including the correct folder and “.rds” extension.
+check_brmsfit_file <- function(file, .check_folder = TRUE) {
+
```

Comment:
you may want to reduce the amount of empty lines here but this is really just minor style aspect